test/Core.Test/Services/OrganizationServiceTests.cs

```diff
@@ -962,4 +1018,88 @@ public async Task Delete_Success(Organization organization, SutProvider<Organiza
         await organizationRepository.DidNotReceiveWithAnyArgs().DeleteAsync(default);
         await applicationCacheService.DidNotReceiveWithAnyArgs().DeleteOrganizationAbilityAsync(default);
     }
+
+    [Theory, BitAutoData]
+    public async Task RevokeUser_Success(Organization organization, [OrganizationUser(OrganizationUserStatusType.Confirmed, OrganizationUserType.Owner)] OrganizationUser owner,
+        [OrganizationUser] OrganizationUser organizationUser, SutProvider<OrganizationService> sutProvider)
+    {
+        sutProvider.GetDependency<IOrganizationRepository>().GetByIdAsync(organization.Id).Returns(organization);
+        sutProvider.GetDependency<IOrganizationRepository>().GetByIdAsync(organizationUser.OrganizationId).Returns(organization);
+        sutProvider.GetDependency<ICurrentContext>().OrganizationOwner(organization.Id).Returns(true);
+        sutProvider.GetDependency<ICurrentContext>().ManageUsers(organization.Id).Returns(true);
+        var organizationUserRepository = sutProvider.GetDependency<IOrganizationUserRepository>();
+        organizationUserRepository.GetManyByOrganizationAsync(organizationUser.OrganizationId, OrganizationUserType.Owner)
+            .Returns(new[] { owner });
+        var eventService = sutProvider.GetDependency<IEventService>();
```

Comment:
This setup looks identical for the following 4 tests. If that's correct, please extract to `RestoreRevokeUser_Setup` or similar.